### ISSUE TITLE
Links from Discord Open in Browser Instead of Redomi App

### ISSUE THREAD

**Issue by ItsTatsuya** [acszo/Redomi#15]
### Describe the bug

When converting and opening music links shared on Discord using the Redomi app, the links open in the web browser instead of the Redomi app. This issue is specific to links from Discord, while links from other platforms work as expected.

### Expected behavior

The converted music links shared on Discord should open directly in the Redomi app instead of the web browser.

### Device

- Redomi Version: 1.2.3
- Android Version: 14 qpr2
- Discord Version: 232.15

**Comment by ItsTatsuya** [comment]
could be a discord side issue

**Comment by acszo** [comment]
Yeah, it is a known issue on discord (since January iirc), only some apps actually do open in app, like youtube and twitter, but reddit and yt music don't just like redomi for me. [Here](https://discord.com/blog/discord-patch-notes-april-2024) they mentioned "Resolved bugs where certain links to popular websites would open in the in-app browser instead of the relevant applications" but it's still the same, idk, discord moment

**Comment by ItsTatsuya** [comment]
i will write a bug report to discord, i doubt its gonna do anything but worth a shot

**Comment by aryaoffixial** [comment]
Did you tried using Linksheet app? I think setting that as a default browser could open links in selected apps
